### ISSUE TITLE
Can't view some "Hidden" thumbnails

### ISSUE THREAD

**Issue by bluefiberbread** [Darayavaush/decensooru#2]
Using Chrome with the latest version of BBB (8.0.2) and it won't expose the hidden thumbnails. Tampermonkey (4.4) is being used too.

https://danbooru.donmai.us/posts?limit=200&page=5&tags=banned_artist
https://i.imgur.com/2MyKpM6.jpg

**Comment by Darayavaush** [comment]
1. Did you modify the BBB code as specified in the instructions?
2. Did you ever see a text box about the batch pulling in the top right corner? 
3. Is there a message "Nothing to do, wait for next batch." in the console?

**Comment by bluefiberbread** [comment]
Yeah, it breaks/disables BBB too

**Comment by Darayavaush** [comment]
"Breaks"? Do you see the "BBB settings" menu at the top? Could you post the modified BBB source, please?
Also, what about the other two questions?
Also also, I just installed it on Tampermonkey myself and Chrome informed me about a cross-origin request. Did you get this window?

**Comment by bluefiberbread** [comment]
No, I don't see BBB settings in the menu after following the instructions, I got the cross-origin request. I did modify the BBB code as specified in the instructions. 

https://puu.sh/yHoQg/9c00a66710.jpg
https://puu.sh/yHoRx/84bb2e02dd.jpg

**Comment by Darayavaush** [comment]
You modified the second one incorrectly, inserting the new block before the specified line instead of after it. Since it references an undeclared variable, that's enough to break everything.

**Comment by bluefiberbread** [comment]
https://puu.sh/yHDXm/0f333a1b3d.jpg
https://puu.sh/yHDYZ/ed395a3993.jpg

It shows some of the thumbnails (just a few now) but the full picture is still hidden

**Comment by Darayavaush** [comment]
It should be inserted _immediately_ after the specified line, not randomly somewhere below.

**Comment by bluefiberbread** [comment]
I reinstalled Tampermonkey and it works fine now but there are some hidden thumbnails. Where can I PM the links to you

**Comment by Darayavaush** [comment]
df6c020dc48b5743a6ec52aa5551c9b9 (or you can simply list the IDs here)

**Comment by bluefiberbread** [comment]
I sent it to df6c020dc48b5743a6ec52aa5551c9b9

**Comment by Darayavaush** [comment]
Yeah, I see what happened there. I'll think about how it can be fixed.

**Comment by bluefiberbread** [comment]
did you come up with a solution

**Comment by Darayavaush** [comment]
There are several potential solutions, but all of them are going to be a huge PITA to implement and I'm fairly busy. I'll get to it eventually™, don't worry.

**Comment by Zanmato27** [comment]
Any updates on this?

**Comment by Darayavaush** [comment]
Ha, exactly one year later. Nice. :D

Unfortunately, eventually™ has not yet arrived, but as a stopgap measure I pushed an update with all currently hidden old posts (3k of them). This should last us the next year. :)

**Comment by b1116490** [comment]
Hello, I'm hijacking this thread because I have the same issue as OP before, Have the 

> Decensooru started. There are 1 items in DB 
Nothing to do, wait for next batch.

messages in the console and no thumbnails or pictures even on old art. I am 99.9% sure I did everything correctly. I use tampermonkey by the way.

**Comment by Darayavaush** [comment]
Your issue is completely different, but whatever. I just added an option to do a full refresh. Update the script and look into the footer of any page on Danbooru. Click 'force full refresh' _once_ and tell me what happens.

**Comment by b1116490** [comment]
My bad, I thought it was the same. Anyway, it works now. Thanks!

There are a few entries without art still but for the most part it seems fine.
